data/styles/clear/include/Icons.mapcss

```diff
@@ -2492,50 +2492,59 @@ area|z15-[amenity=sanitary_dump_station]
 
 /* 7.1 Parking */
 
-node|z16[amenity=parking],
-area|z16[amenity=parking]
-{icon-image: parking-s.svg;icon-min-distance: 25;}
-
-node|z17-[amenity=parking],
-area|z17-[amenity=parking]
-{icon-image: parking-m.svg;}
-
+node|z16-[amenity=parking],
+area|z16-[amenity=parking],
+{icon-image: parking-s.svg;icon-min-distance: 5;}
 node|z18-[amenity=parking],
-area|z18-[amenity=parking]
-{font-size: 11;}
-
-node|z16-17[amenity=parking][access=permissive],
-node|z16-17[amenity=parking][access=private],
-{icon-image: zero-icon.svg;}
-
-
-node|z17[amenity=parking_entrance],
-area|z17[amenity=parking_entrance]
+area|z18-[amenity=parking],
+{icon-image: parking-m.svg;font-size: 11;icon-min-distance: 10}
```

Comment:
for regular and paid parkings please use the -m icon from z17-

otherwise at z17 both regular and street side parkings will looks the same with the -s icon

Reply:
even bicycle parking uses -m at z17 :)